docs/api.md

```diff
@@ -118,8 +121,46 @@ type AddonsSpec struct {
 }
 
 // TODO maybe we have profiles for scaling behaviors
-type AutoScalerAddonSpec struct {
-    ResourceLimits LimitRange
+type ClusterAutoscaling struct {
+	// Maximum number of nodes in all node groups.
+	// Cluster autoscaler will not grow the cluster beyond this number.
+	// +kubebuilder:validation:Minimum=0
+	MaxNodesTotal int32 `json:"maxNodesTotal"`
+
+	// Should CA scale down the cluster
+    // default: true
+	ScaleDownEnabled bool `json:"enabled,omitempty"`
```

Comment:
I propose we remove this field and have a followup discussion about whether there's some implied policy object to drive things like turning the entire system on or off, etc.

Reply:
that makes sense to me. i think we could easily just drop this and have scale down always enabled, and happy to have the followup conversation.

Reply:
Sounds like we agreed to remove this but it remains

Reply:
i just haven't pushed a new version yet. was waiting to gather a few more responses.